playbooks/adhoc/preflight/check.yml

```diff
@@ -0,0 +1,32 @@
+---
+- hosts: OSEv3
+  gather_facts: no
+  roles:
+    - openshift_preflight/facts
```

Comment:
Instead of nesting these roles under openshift_preflight, they should really be broken out (openshift_preflight_facts, openshift_preflight_all, openshift_preflight_masters, openshift_preflight_nodes, openshift_preflight_verify, etc)